### ISSUE TITLE
Delay sending OA notification email until publication has been in RMD for at least 1 month

### ISSUE THREAD

**Issue by anaelizabethenriquez** [psu-libraries/researcher-metadata#615]
Now that we are sending emails weekly, there is the possibility that we'll send an email about an item when it is extremely new to RMD. (It is only a possibility, not a guarantee, because of the six-month delay between emails to a given researcher and because of the cap on the number of emails sent weekly.)

Ideally, we'd like to give the OAB importer for items without DOIs (#612) time to run on a new publication before emailing the author about it. As of November 2022, that import takes two weeks to run. If we wait at least a month, we'll know it's had time to run.

**Comment by ajkiessl** [comment]
@anaelizabethenriquez Sorry, I may have forgotten to mention this to you, but I ran the "new" import for OAB a week or two ago and that ran to completion.  The "new" import is not split up by DOI/no DOI.  It just does a DOI search or title search depending on whether or not a new publication has a DOI.  Therefore, we are caught up with OAB checks for the publications without DOIs.  The "new" imports for OAB and Unpaywall will continue to run Sunday the day before the emails go out, and should only have a couple hundred records to get through each week (a small enough amount to catch us up before emails go out).  

I just split up the full import (in #612) we had running weekly that iterates through all publications.  It was basically doing what the "new" import does (does a title search if not DOI or a DOI search if DOI), but was doing it for _all_ the publications.  This was taking too long and was prone to error, so I split it up.  

We could consider not doing the full import for publications without DOIs in the future.  It may not be necessary to continue to do title searches for records that have already been checked and nothing was found.  However, there's the chance something was changed or recently added to OAB, so the full import could catch that stuff.
